client/core/core.go

```diff
@@ -2028,9 +2039,29 @@ func (c *Core) handleReconnect(uri string) {
 		log.Errorf("unable to authorize DEX at %s: %v", uri, err)
 		return
 	}
-	details := fmt.Sprintf("DEX at %s has reconnected", uri)
-	c.notify(newSystemNote("DEX reconnect", details, db.Success))
-	log.Infof(details)
+}
+
+// handleConnectEvent is called when a WsConn indicates that a connection was
+// lost or established.
+//
+// NOTE: This handler is only used to post notifications to the user.
```

Comment:
"only use to post notifications"   But it also set sets each `dexConnection.connected` flag.  And I think that is good because dexConnection can do things differently if it believes to be disconnected.